assets/airports/eddf.json

```diff
@@ -1911,36 +1911,36 @@
                 ["KERAX", "GED", "GEDSI", "MTR", "ETARU", "DF951", "IBLUS"]
             ]
         },
-        "KERAX3B": {
-            "icao": "KERAX3B",
-            "name": "Kerax Three Bravo",
+        "KERAX4B": {
```

Comment:
As with SIDs, STAR declarations need to have the same name as the ICAO name. so this should be KERA4B. Same applies to other STARs

Reply:
Will change both within the next few hours. Thanks!